src/controllers/confirm.company.controller.ts

```diff
@@ -77,6 +83,11 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
   }
 };
 
+const companyHasNoDirectors = async (companyNumber: string) => {
+  const companyMetrics: MetricsApi = await getCompanyMetrics(companyNumber);
+  return companyMetrics?.counts?.appointments?.activeDirectorsCount == 0;
```

Comment:
Does the API list these as mandatory return fields? Could throw an undefined error here if not.

Reply:
Discussed and agreed that these fields are not marked as optional on api-sdk-node, but any null fields would not cause an error here.